app/integration_action.go

```diff
@@ -375,10 +375,10 @@ func (w *LocalResponseWriter) WriteHeader(statusCode int) {
 }
 
 func (a *App) doPluginRequest(c *request.Context, method, rawURL string, values url.Values, body []byte) (*http.Response, *model.AppError) {
-	return a.ch.doPluginRequest(c, method, rawURL, values, body)
+	return a.ch.srv.pluginService.doPluginRequest(c, method, rawURL, values, body)
```

Comment:
why not pass the pluginService to the `App`? The same way we have `ch`? Law of Demeter comes to mind

Reply:
This is a great point, thanks @iomodo. This is probably going to be our next step which would have an `App` struct with products. At this point, by the lifecycle, we are treating plugins like a product which makes sense.